jpa/eclipselink.jpa.test.jse/src/it/java/org/eclipse/persistence/jpa/test/query/TestQueryCase.java

```diff
@@ -328,8 +572,125 @@ public void testQueryCase3() {
         }
     }
 
+//    @Test
```

Comment:
is this intentional?

Reply:
Yes, these were intentional. I commented out these 2 tests to disable them because they fail for an unrelated failure. I wrote the tests, and I believe they are valid tests, but they currently fail. I chose to only comment them out because I did not want to lose the test. I think using @Ignore is a much better idea though (I forgot that exists) and I will update to use that here instead.